src/sql/platform/theme/common/colorRegistry.ts

```diff
@@ -41,4 +41,19 @@ export const gradientOne = registerColor('gradientOne', { light: '#f0f0f0', dark
 export const gradientTwo = registerColor('gradientTwo', { light: gradientTwoColorOne, dark: gradientTwoColorTwo, hc: gradientTwoColorTwo }, nls.localize('gradientTwo', "The bottom color for the banner image gradient"));
 export const gradientBackground = registerColor('gradientBackground', { light: '#fff', dark: 'transparent', hc: 'transparent' }, nls.localize('gradientBackground', "The background color for the banner image gradient"));
 
-
+// --- Notebook Colors
+export const toolbarBackground = registerColor('toolbarBackground', { light: '#F5F5F5', dark: '#252423', hc: '#000000' }, nls.localize('toolbarBackground', "Markdown toolbar background"));
+export const toolbarForeground = registerColor('toolbarForeground', { light: '#323130', dark: '#FFFFFe', hc: '#FFFFFe' }, nls.localize('toolbarForeground', "Markdown toolbar icons"));
```

Comment:
is this string correct here and the bottom border below?

Reply:
Yes, string is correct.